amino.go

```diff
@@ -331,7 +331,7 @@ func (cdc *Codec) UnmarshalBinaryLengthPrefixedReader(r io.Reader, ptr interface
 	}
 	l = int64(u64)
 	if l < 0 {
-		_ = errors.Errorf(
+		_ = errors.Errorf( //nolint:errcheck
```

Comment:
what is this?)

Reply:
I was looking at this as well, with amino being a bit touchy I didn't want to change to much stuff in this PR so I left it

Reply:
Right. But this code does not do anything, so maybe we should comment it completely.